Sources/PointFree/Pricing.swift

```diff
@@ -109,10 +110,11 @@ public enum Pricing: Codable, DerivePartialIsos {
 }
 
 let pricingResponse =
-  writeStatus(.ok)
-    >-> respond(pricingView)
+  redirectCurrentSubscribers
+    <| writeStatus(.ok)
+    >-> respond(pricingView.contramap(lower))
 
-private let pricingView = View<(Pricing, Database.User?, Route)> { pricing, user, currentRoute in
+private let pricingView = View<(Database.User?, Pricing, Route)> { user, pricing, currentRoute in
```

Comment:
re-arranged order of tuple so that the middleware transformer could operator on `T2<User?, A>`, but could have just as easily made the transformer operate on `T3<A, User?, B>`...

Reply:
Yeah, I've found keeping `User` first helps make user-based middleware more reusable. I wonder though if using `over{n}` means it doesn't matter.

Reply:
that's true, we should be able to write transformers on `T2<A, B>`, and just use some kind of `over` or some new combinator to lift existing middleware into that world.